packages/spaces/src/lib/SpacesLink/spaces-link-types.tsx

```diff
@@ -31,24 +33,24 @@ export type SpacesLinkProps = {
   /** Children can be a react child or render prop. */
   children?: JSX.Element | ((props: any | undefined) => JSX.Element);
   /** Tag to overwrite the root component rendered. */
-  tag?: string;
+  tag?: ElementType<any, keyof JSX.IntrinsicElements>;
   /** Tag to overwrite the body component that renders the title, description and data values.
    * It defaults to CardBody or div depending on the value of the variant prop.
    */
-  bodyTag?: string;
+  bodyTag?: ElementType<any, keyof JSX.IntrinsicElements>;
   /** Tag to overwrite the title component. If variant prop is set to "card", defaults to CardTitle.
    * If variant is set to "list", defaults to ListItemHeading. Overwise, defaults to div.
    */
-  titleTag?: string;
+  titleTag?: ElementType<any, keyof JSX.IntrinsicElements>;
   /** Tag to overwrite the text component. If variant prop is set to "card", defaults to Card Text.
    * If variant is set to "list", defaults to ListItemText. Otherwise, defaults to div.
    */
-  textTag?: string;
+  textTag?: ElementType<any, keyof JSX.IntrinsicElements>;
   titleClassName?: string;
   /** When true, utilizes the Card component for styling. */
   card?: boolean;
   /** When true, renders an @availity/mui-icon next to the title if present on the Space. */
-  icon?: boolean;
+  icon?: ({ ...props }: SvgIconProps) => JSX.Element;
```

Comment:
I don't think the spreading of props is doing anything here

Reply:
You're right, that was copy pasta. I'll get that fixed.